Building/Building/doc/spec.md

```diff
@@ -0,0 +1,207 @@
+# Building
+
+## Description
+
+This entity contains a harmonised description of a Building. This entity is 
+associated with the vertical segments of smart homes, smart cities, industry and
+related IoT applications.
+
+This data model has been partially developed in cooperation with mobile
+operators and the [GSMA](http://www.gsma.com/connectedliving/iot-big-data/),
+compared to GSMA data model following changes are introduced:
+
+* the reference to `BuildingType` is removed, since `BuildingType` compared
+  to `category` attribute does not introduce significant information.
+
+* `category` attribute is required.
+
+* `openingHours` is introduced following schema.org data model to allow
+  fine-grained on building opening times. GSMA supported this as free text
+  in the `notes` attribute (remove as well).
+
+* `refSubscriptionService` is not supported, since `SubscriptionService` model
+  is not supported currently.
+
+## Data Model
+
+For a full description of the following attributes refer to GSMA
+[IoT Big Data Harmonised Data Model](https://www.gsma.com/iot/wp-content/uploads/2016/11/CLP.26-v1.0.pdf)
+
```

Comment:
we need to update this reference to point to the latest version of the spec

Reply:
where can i find it?

Reply:
the link may be wrong actually, this one should the correct one: https://www.gsma.com/iot/wp-content/uploads/2016/06/CLP.26-v4.0.pdf but i couldn't find anything newer

Reply:
it should be this one, the path ( 2016 ) does not correspond to the pdf data (december 2017). i noticed that some models seems changed (e.g. AirQuality) should be this a concern for us?